packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -633,7 +633,7 @@ export interface DomainProps {
   /**
    * The minimum TLS version required for traffic to the domain.
    *
-   * @default - TLSSecurityPolicy.TLS_1_0
+   * @default - no TLS security policy set, allowing the service to use its default policy
```

Comment:
This needs to be updated to say TLS 1.2 since we have decided to keep the default in CDK